### PR TITLE
Updated the DNE side, to enable vcenter cluster working

### PR DESCRIPTION
**Pull request by kushagra-mahajan**
- [X] I have read the [CONTRIBUTION GUIDE][contributing]
- [ ] This is a bug
- [X] This is a feature request
- [ ] This is a syntax/style fix
- [ ] This is an improvement to code quality

---

Provide a clear description of changes:
 
[contributing]: http://dellemc-symphony.readthedocs.io/en/latest/contributingtosymphony.html

### CODE REVIEW

File: dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/domain/CredentialDetails.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+/**
+ * TODO: Document Usage
+ * <p/>
```
Comment (giddyelysium) [inline dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/domain/CredentialDetails.java]: There are some JavaDoc syntax issues in your code please take another look. In this case, it relates to the HTML content and explicitly the lack of closing or duplicate opening/closing parenthesis.

File: dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/domain/EndpointDetails.java [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * TODO: Document Usage
+ * <p/>
```
Comment (giddyelysium) [inline dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/domain/EndpointDetails.java]: There are some JavaDoc syntax issues in your code please take another look. In this case, it relates to the HTML content and explicitly the lack of closing or duplicate opening/closing parenthesis.

File: dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/domain/CredentialDetails.java (hunk 1 above)
Comment (kushagra-mahajan) [inline dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/domain/CredentialDetails.java]: Done!

File: dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/domain/EndpointDetails.java (hunk 2 above)
Comment (kushagra-mahajan) [inline dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/domain/EndpointDetails.java]: Done!

**Pull request merged**